code/Justin/Python/lab_2a.py

```diff
@@ -0,0 +1,8 @@
+num = input('Enter a number: ')
+animal = input('Enter your favorite animal: ')
+state = input('Enter a state: ')
+vehicle = input('Enter a vehicle: ')
+movie = input('Enter your favorite movie: ')
+length = input('Enter a number: ')
+
+print('When I was on a trip to ' + state.title() + ' I randomly came across a ' + animal + '. They asked for a ride, so I took them to see ' + movie.title() + '. The theater was ' + num + ' hours away, so after the ride and movie we spent around ' + length + ' hours together.')
```

Comment:
Good work! F-string was not required for this assignment, but consider using it next time, it would have made this more readable.